test/prototype/inductor/test_int8_sdpa_fusion.py

```diff
@@ -157,7 +157,7 @@ def _check_common(
     )
     @config.patch({"freezing": True})
     def _test_sdpa_int8_rewriter(self):
-        from torch.export import export_for_training
+        from torch.export import export
```

Comment:
If this test covers fp8, we'd better rename it.

Reply:
Thanks and modified.